src/main/resources/db/migration/V3__Create_job_roles_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE job_roles (
+    jobRoleId INT PRIMARY KEY,
+    roleName VARCHAR(255) NOT NULL,
+    location VARCHAR(255),
```

Comment:
max 25? check the longest town/city name in the word ;)

Reply:
Fixed - changed to enum